### DISCUSSION TITLE
Regarding socketIO communication, is there a way to hide the room name when transmitting binary data?

### DISCUSSION THREAD

**Discussion by luoyu-123** [miguelgrinberg/Flask-SocketIO#1739] [Q&A]
This is what I am using now:

![image](https://user-images.githubusercontent.com/59160360/143230114-509027b6-caa5-4c8f-9e88-01f8bff904da.png)

This is what i expected：
![image](https://user-images.githubusercontent.com/59160360/143230312-db85872c-f089-42c9-bc38-961dd29ef6e9.png)

How do I implement it in the flask-socketio package?

Best wishes！

**Comment by miguelgrinberg** [comment]
Socket.IO does not work like that. If you need absolute control of what goes on the wire, then use WebSocket directly.

**Reply by luoyu-123** [reply]
thank you
